README.md

```diff
@@ -7,13 +7,41 @@
 
 ---
 
-* **[Installation](#installation)**
-* **[Setup](#setup)**
-* **[Usage](#usage)**
-* **[API](#api)**
-* **[License](#license)**
-
 
+<!-- vim-markdown-toc GFM -->
+
+* [Installation](#installation)
+* [Setup](#setup)
+* [Usage](#usage)
+  * [Usage with File Config](#usage-with-file-config)
+* [API](#api)
+  * [LogDNAHandler(key, [options])](#logdnahandlerkey-options)
+    * [key](#key)
+    * [options](#options)
+      * [app](#app)
```

Comment:
Detailing each option in a TOC is maybe too much :)

Reply:
Ah. Sure. Its generated from headers. will fix